packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -1,5 +1,5 @@
-@import '../styles/placeholder.scss'; // Contains the placeholder mixin
-@import '../styles/mixins';
+@import '~@automattic/calypso-color-schemes';
```

Comment:
Need to beware not to import this multiple times as this creates styling duplication.

Reply:
Hm! Wouldn't it now be included multiple times by importing this to Gutenboarding, which also includes this package? 🤔 

Should it be declared it as `peerDependencies`?

Reply:
> Wouldn't it now be included multiple times by importing this to Gutenboarding, which also includes this package?

I think this is happening right now. If you look at `entry-gutenboarding.css` file the `:root` selector where all the color declarations are repeated at least twice. Need to investigate how this is happening but not in this PR.

> Should it be declared it as `peerDependencies`?

Not sure if `sass-loader` actually reads these things.